jsonrpc/src/methods/beacon.json

```diff
@@ -200,5 +200,17 @@
     "result": {
       "$ref": "#/components/contentDescriptors/GossipResult"
     }
+  },
+  {
+    "name": "portal_beaconGetContent",
+    "summary": "Get content from the local database if it exists, otherwise look up the target content key in the network.",
```

Comment:
I do think we need to somewhere explicitly specify that the content needs to get verified by the RPC server and throw an error if verification fails.

Else we will end up with differences between clients like we have now for the `portal_*RecursiveFindContent` method (which as @morph-dev mentioned, we should clearify also, but perhaps in another PR)

Reply:
Okay, I'll add more detail to the description in that case